core/src/main/scala/org/locationtech/rasterframes/ref/RasterRef.scala

```diff
@@ -74,10 +75,16 @@ object RasterRef extends LazyLogging {
       ProjectedRasterTile(rr.realizedTile.convert(ct), extent, crs)
   }
 
+  val embeddedSchema: StructType = StructType(Seq(
+    StructField("source", RasterSourceType.sqlType, true),
+    StructField("bandIndex", IntegerType, false),
+    StructField("subextent", schemaOf[Extent], true),
+    StructField("subgrid", schemaOf[GridBounds], true)
+  ))
+
   implicit val rasterRefSerializer: CatalystSerializer[RasterRef] = new CatalystSerializer[RasterRef] {
-    val rsType = new RasterSourceUDT()
     override val schema: StructType = StructType(Seq(
-      StructField("source", rsType.sqlType, false),
+      StructField("source", RasterSourceType, true),
```

Comment:
thinking aloud, will this affect the python UDT?

Reply:
No.... this is in weeds of a parquet serialization edge case I discovered on accident. The rest of Catalyst treated these as synonyms.